cmake/external/triton.cmake

```diff
@@ -44,13 +44,11 @@ if (WIN32)
   vcpkg_install(re2)
   vcpkg_install(boost-interprocess)
   vcpkg_install(boost-stacktrace)
-  vcpkg_install(zlib)
   vcpkg_install(pthread)
   vcpkg_install(b64)
 
   add_dependencies(getb64 getpthread)
```

Comment:
They all need be added to ThirdPartyNotices.txt since the EP you built will contains code from the 3rd-party libraries.

Reply:
it is a removal.